quickbooks_endpoint.rb

```diff
@@ -18,6 +18,7 @@ class QuickbooksEndpoint < EndpointBase::Sinatra::Base
 
   post '/get_tokens' do
     base_service = QBIntegration::Service::Token.new(@config)
+    base_service.access_token.refresh!
```

Comment:
Are you sure the access_token method isnt already refreshing it ?

Reply:
I removed the call for refresh in `Auth#access_token`

Reply:
Actually, this part of the code might be wrong. When I started checking the history of the transactions for get tokens, I noticed that the refresh token and access token never change. It makes sense that the invalid grant happens if thats the case. I think the access token is immutable and `refresh!` returns a new access token that we lose.  I'll make a new PR for it in the develop branch